cocos/ui/UIEditBox/UIEditBox.h

```diff
@@ -254,16 +262,124 @@ namespace ui {
         bool initWithSizeAndBackgroundSprite(const Size& size,
                                              const std::string& normal9SpriteBg,
                                              TextureResType texType = TextureResType::LOCAL);
-            
-        
+
         /**
          * Init edit box with specified size. This method should be invoked right after constructor.
          * @param size The size of edit box.
          * @param normal9SpriteBg  background image of edit box.
          * @return Whether initialization is successfully or not.
          */
+        CC_DEPRECATED_ATTRIBUTE
```

Comment:
what's the reason to mark this function as __DEPRECATED__, I don't think it's out-of-date

Reply:
All of other controls which has three-state texture, such as ui::Button, ui::CheckBox, ui::RadioButton, etc, don't have a create or init function with Scale9Sprite parameters, just use std::string to specify the texture soures.

for unifying, I mark these as DEPRECATED

Reply:
I think EditBox is a special one when developers use EditBox, most of them don't care its three-state texture, but for Button most care. 

suggest to remove those DEPRECATED mark, and then `macro block` inside EditBox::create() function.